### PR TITLE
DSPDC-915 Port example workflow to reusable template.

### PR DESCRIPTION
**Pull request by danxmoran**
Depends on broadinstitute/monster-helm#18

### CODE REVIEW

**Comment by danxmoran** [comment]
Proof! https://argo.monster-dev.broadinstitute.org/workflows/encode/transfer-s3-to-gcs-9s8v6

File: orchestration/Chart.yaml [hunk 1]
```diff
@@ -7,6 +7,6 @@ type: application
 version: 0.0.0
 dependencies:
   - name: argo-templates
-    version: ~0.0.1
+    version: 0.0.0-DSPDC-915
```
Comment (danxmoran) [inline orchestration/Chart.yaml:10]: TODO: Set to `0.0.3`

File: orchestration/Chart.lock [hunk 2]
```diff
@@ -0,0 +1,6 @@
+dependencies:
```
Comment (danxmoran) [inline orchestration/Chart.lock:1]: TODO: Remove this

File: orchestration/templates/ingest-encode-data.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: WorkflowTemplate
 metadata:
-  name: ingest-clinvar-archive
+  name: ingest-encode-data
```
Comment (raaidbroad) [inline orchestration/templates/ingest-encode-data.yaml:4]: YUGE CHANGE

File: orchestration/templates/serviceaccount.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    rule: RunAsAny
+  volumes:
+    - '*'
+---
+# Permissions needed by a k8s SA to run the ClinVar workflow.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .k8sName }}-role
+rules:
```
Comment (raaidbroad) [inline orchestration/templates/serviceaccount.yaml:33]: are these rules intentionally different than what was in the deleted role file?

Comment (danxmoran) [reply orchestration/templates/serviceaccount.yaml:33]: Yes, the role file was over-provisioning permissions (I think we copy-pasted it from the Argo quickstart)

**Pull request merged**